Dockerfile

```
@@ -16,6 +16,13 @@ ARG CISA_USER="cisa"
 ENV CISA_HOME="/home/${CISA_USER}"
 ENV VIRTUAL_ENV="${CISA_HOME}/.venv"
 
+
+# Versions of the Python packages installed directly
+ENV PYTHON_PIP_VERSION=23.0
+ENV PYTHON_SETUPTOOLS_VERSION=67.3.1
+ENV PYTHON_WHEEL_VERSION=0.38.4
+ENV PYTHON_PIPENV_VERSION=2023.2.4
```

Might as well alphabetize these suckas:
```suggestion
ENV PYTHON_PIP_VERSION=23.0
ENV PYTHON_PIPENV_VERSION=2023.2.4
ENV PYTHON_SETUPTOOLS_VERSION=67.3.1
ENV PYTHON_WHEEL_VERSION=0.38.4
```

---

That's fair. I'd originally ordered them intentionally by order of use but considering how often we beat the drum about alphabetizing unless there's a technical reason we cannot do so it was a bit silly of me 😅

---

Haha, I had a feeling that was what you had been thinking!